### PR TITLE
Updated config maps for grafana and influxdb

### PR DESCRIPTION
**Pull request by sudeeshjohn**
1.  Updated deployment guide for mtu changes
2. Updated and added ConfigMap for Grafana dashboard
3. Updated Grafana ingress yaml
4. Added new ConfigMap for influxdb for creating databases

### CODE REVIEW

**Comment by ltccci** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/ppc64le-cloud/test-infra/pull/181#" title="Author self-approved">sudeeshjohn</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign **mkumatag** after the PR has been reviewed.
You can assign the PR to them by writing `/assign @mkumatag` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=ppc64le-cloud%2Ftest-infra).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/ppc64le-cloud/test-infra/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by ltccci** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/ppc64le-cloud/test-infra/pull/181#issuecomment-826576274" title="Approved">mkumatag</a>*, *<a href="https://github.com/ppc64le-cloud/test-infra/pull/181#" title="Author self-approved">sudeeshjohn</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=ppc64le-cloud%2Ftest-infra).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/ppc64le-cloud/test-infra/blob/master/OWNERS)~~ [mkumatag]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by Rajalakshmi-Girish** [comment]
> 
> 
> /approve
> /hold for @Rajalakshmi-Girish to review

As new to grafana and influxdb, need sometime to understand and review.

**Comment by Rajalakshmi-Girish** [comment]
> 
> 
> /approve
> /hold for @Rajalakshmi-Girish to review

I tried to see the grafana related specs upstream and understand what is intended. But I find it difficult to follow.
@mkumatag @sudeeshjohn is there any reference for me, please.

File: Deployment Guide.md [hunk 1]
```diff
@@ -38,6 +38,9 @@ $ cd cluster
 $ kubectl apply -f .
 ```
 
+8. Change the calico MTU to 1420 (This is for Jenins agents to work properly)
+   Folllow, https://cloud.ibm.com/docs/containers?topic=containers-kernel#calico-mtu and change the mtu to 1420
+
```
Comment (mkumatag) [inline Deployment Guide.md:43]: I'm not really comfortable with it, will have to figure out some other way to fix it. Can be a new k8s cluster in powervs that matches MTU with IKS,?!

File: config/prow/grafana-dashboard/influxdb_pv.yaml [hunk 2]
```diff
@@ -6,8 +6,9 @@ metadata:
   name: influxdb
   namespace: grafana-dashboard
 spec:
+  storageClassName: ibmc-vpc-block-retain-10iops-tier
```
Comment (mkumatag) [inline config/prow/grafana-dashboard/influxdb_pv.yaml:9]: why not default SC? is it because to retain the volume?

**Review by mkumatag — CHANGES_REQUESTED** [review]

**Comment by ltccci** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/ppc64le-cloud/test-infra/pull/181#" title="Author self-approved">sudeeshjohn</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign **mkumatag** after the PR has been reviewed.
You can assign the PR to them by writing `/assign @mkumatag` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=ppc64le-cloud%2Ftest-infra).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/ppc64le-cloud/test-infra/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: config/prow/grafana-dashboard/influxdb_pv.yaml (hunk 2 above)
Comment (sudeeshjohn) [reply config/prow/grafana-dashboard/influxdb_pv.yaml:9]: yes, this is for retaining the volume

File: Deployment Guide.md (hunk 1 above)
Comment (sudeeshjohn) [reply Deployment Guide.md]: Removed this part

File: config/prow/grafana-dashboard/influxdb_deployment.yaml [hunk 3]
```diff
@@ -22,13 +22,21 @@ spec:
...
+              name: init-scripts
       restartPolicy: Always
       volumes:
         - name: var-lib-influxdb
           persistentVolumeClaim:
             claimName: influxdb
+        - configMap:
+            name: init-scripts
+          name: init-scripts
+
```
Comment (mkumatag) [inline config/prow/grafana-dashboard/influxdb_deployment.yaml:42]: can you delete this empty line

**Review by mkumatag — CHANGES_REQUESTED** [review]

**Review by mkumatag — APPROVED** [review]
/lgtm

**Pull request merged**